projects/odf_transform/odf_transform/config.json

```diff
@@ -28,5 +28,11 @@
         "keywords_vocabulary": "GCMD Science Keywords",
         "Conventions": "CF-1.6,CF-1.7,ACDD1.1",
         "comment": ""
+    },
+    "flag_convention": {
+        "default": {
+            "flag_values": [0, 1, 2, 3, 4, 9],
+            "flag_meaning": "not_evaluated correct inconsistent_with_other_values doubtful erroneous modified missing"
```

Comment:
Not sure if this is used, but seems like `flag_meaning` should be a list and have the same number of elements as flag_value

Reply:
it is if not than I made a mistake

Reply:
I just added the missing flag 5

Reply:
but `flag_meaning` is still string not a list